### ISSUE TITLE
metadata platform version apparently must be a floating point

### ISSUE THREAD

**Issue by mattray** [chef/cookstyle#658]
Within my metadata.rb I had
```
supports 'suse', '>= 12'
```
which worked for building Policyfiles, but when I tried to upload it to Hosted Chef it complained about 
```
$ knife cookbook upload systemd_timesyncd
Uploading systemd_timesyncd [0.3.0]
ERROR: The data in your request was invalid
Response: Invalid value '>= 12' for metadata.platforms
```
Changing the line to
```
supports 'suse', '>= 12.0'
```
fixes the issue and allows the cookbook to be uploaded.

### What category of cop is this?:
ChefCorrectness

### What it would autocorrect to if applicable
Convert from an integer to a float?

**Comment by tas50** [comment]
This seems like a Chef Infra Server bug

**Comment by tas50** [comment]
Shipped in 6.13
